docs/fsharp/language-reference/members/constructors.md

```diff
@@ -82,9 +82,11 @@ The following version of the previous code illustrates the combination of ordina
 
 [!code-fsharp[Main](../../../../samples/snippets/fsharp/lang-ref-2/snippet3507.fs)]
     
+## Constructors in inherited class
+You can definition constructors combined inherited class. For more information, see [Constructors and inheritance](../inheritance.md#constructors-and-inheritance).
```

Comment:
That sounds great.  @kekyo, if you could change to what @svick suggested (assuming you're okay with the change), then I'll happily merge this PR.